content/master/concepts/managed-resources.md

```diff
@@ -419,6 +425,22 @@ does not create the external resource until the referenced object is ready. In
 this example, creation call of Azure MySQL Server will not be made until
 referenced `ResourceGroup` has its `status.condition` named `Ready` to be true.
 
+## Management Policies
+
+Crossplane offers a set of management policies that allow you to define the
+level of control it has over external resources. You can configure these
+policies using the `spec.managementPolicy` field in the managed resource
+definition. The available policies include:
+
+- `FullControl (Default)`: With this policy, Crossplane fully manages and
+controls the external resource.
+- `ObserveOnly`: With the ObserveOnly policy, Crossplane only observes the
+external resource without making any changes or deletions.
```

Comment:
Just a comment to confirm.

So we are agreed on dropping OrphanOnDelete and keeping the DeletionPolicy. Im OK with that no matter what is the end result of the ignore changes proposal.